site/en/docs/webstore/i18n/index.md

```diff
@@ -1,132 +1,124 @@
 ---
 layout: "layouts/doc-post.njk"
-title: Internationalizing Your App
+title: Internationalize your extension
 date: 2017-08-30
-description: How to internationalize your Chrome Web Store app.
+updated: 2021-10-29
+description: How to internationalize your Chrome Web Store extension.
 ---
 
-Unless your app is strictly local in scope, you should _internationalize_ it, making it easy to
+This page describes how to support multiple languages of an extension in the Chrome Web Store.
+
+Unless your extension is strictly local in scope, you should _internationalize_ it, making it easy to
 adapt to various languages and regions. You can then _localize_ it—translate and otherwise adapt it
 so that it works well in a particular locale.
 
-You can internationalize your app even if it initially supports just one locale. For example, you
-might initially publish your app in English (locale code: "en"). Then, after a few weeks or months,
-you might add support for additional locales such as French (locale code: "fr") and Arabic (locale
-code: "ar").
-
 {% Aside %}
 The word _internationalization_ is often abbreviated to _i18n_, referring to the 18
 letters between the letters _i_ and _n_. _Localization_ is sometimes abbreviated to _l10n_.
 {% endAside %}
```

Comment:
```suggestion
{% Aside %}

The word _internationalization_ is often abbreviated to _i18n_, referring to the 18 letters between
the letters _i_ and _n_. _Localization_ is sometimes abbreviated to _l10n_.

{% endAside %}
```

This is more personal preference than a requirement, but I tend to prefer a line break after the opening tag and before the closing tag. This makes it easier for the tooling I use to wrap text at 100 characters. Without these line breaks the open and close aside tags are treated as part of the text being wrapped.

Rewrap at 100 characters.

Reply:
ACK